### PR TITLE
Styles: Typography

### PR DESCRIPTION
**Pull request by varffs**
- Adds basic typography styles.
- Doesnt set them exactly according to design as point values very very large when converted to pixels. Adjusted down by eye.
- Keeps quantity of type sizes down to our boiler 6 sizes
- Adds base of responsive values

### CODE REVIEW

**Comment by mikeyrayvon** [comment]
i dont think the rendering of type sizes is consistent btw adobe software and browsers

**Comment by varffs** [comment]
Points have some relation to pixel density etc, there is a formular. It is a thing some people thing is real but basically it is just them reading the values in their design app and not really understanding them. I used this reference: http://www.endmemo.com/sconvert/pixelpoint.php

**Comment by mikeyrayvon** [comment]
we're missing a size for the kickstarter clock

**Comment by mikeyrayvon** [comment]
probably better we size them visually.  to stay ttrue to their design visually, rather than their numbers

**Comment by varffs** [comment]
Yeah I left the countdown size for when that feature is built and styled, but could in the branch

File: src/styl/site.styl [hunk 1]
```diff
@@ -28,15 +28,18 @@ normalize()
 // TYPOGRAPHY
 
 @import '../../globie-library/type'
+@import 'fonts/aileron'
+
+$font-sans = 'Aileron Regular'
```
Comment (mikeyrayvon) [inline src/styl/site.styl]: should add `font-stack('Arial')` after Aileron as fallback

**Pull request merged**